### DISCUSSION TITLE
Invoice filename changes?

### DISCUSSION THREAD

**Discussion by fyrfli** [kimai/kimai#3042] [Questions & Answers] [answered]
I searched through the issues and discussions and only saw a few references to this: Is it possible to configure the  filename of the generated invoices without changing the core code? (That would mean making that change in the code every release).

**Comment by kevinpapst** [comment, marked as answer]
No, unfortunately  it's not yet possible. 
It's on my roadmap, but please don't ask for a date 😁 

If you want to customize it, see this class: https://github.com/kevinpapst/kimai2/blob/master/src/Invoice/InvoiceFilename.php

**Reply by fyrfli** [reply]
This gets overwritten with a new release, correct?

**Reply by kevinpapst** [reply]
Yeah, it will be overwritten. I want to make it configurable through the invoice templates.

**Comment by fyrfli** [comment]
Thanks for the quick response. :)
